signature-proof-spec/spec.md

```diff
@@ -0,0 +1,329 @@
+# Abstract:
+This document describes a different approach to the current verifying mechanisms in verifiable credentials.
+
+# Introduction:
+Currently there are three primary variants of verifiable credentials, all of them have at least one implementation in different stages of production.
+They are:
+1. JSON-LD family with LD-signatures or with BBS+ Signatures that enable Zero knowledge proofs (ZKPs)
+2. JSON with JSON web signatures, in the form of a JSON Web Token (JWT)
+3. ZKP with Camenisch-Lysyanskaya Signatures (ZKP-CL)
+
+Each of these implementation have various pros and cons and this paper will not go into the details of them. Instead, we would refer to the readers to a paper [3] written by Kaliya young, which goes into the specific details of these various implementations.
+
+Although contrary to what the above paper hopes to achieves, what we propose is another flavour of a verifiable credential that is expressed in JSON-LD using linked data proofs that leverage on Merkle hash trees that are less complex (as compared to [debatable]) ZKP implementations and more pragmatic in practice [also debatable] [5]. 
```

Comment:
I would not say it's debatable. ZKP are based on high level mathematics

Reply:
yea ZKP is really complex. But why is it the w3c is converging towards that model instead? I tried looking up for severe limitations regarding our solution, but I cant really find any (maybe I am not looking harder).

Reply:
Our solution is not ZKP at all. With a ZKP credential, you don't need to disclose any information to make assertions.

For instance let's say we want to know If the issuer of a vc is older than 21. With OA you must disclose the info, for instance the birthdate, but you can hide all the rest. With ZKP you can just get the answer to the question

Reply:
👌